app/src/main/java/org/sagebase/crf/fitbit/GsonRequest.java

```diff
@@ -42,185 +42,185 @@
  * @param <T>
  */
 public class GsonRequest<T> extends Request<T> {
-  private static final String LOG_TAG = GsonRequest.class.getCanonicalName();
-
-  /** Charset for request. */
-  private static final String PROTOCOL_CHARSET = "utf-8";
-
-  public static final String DATE_FORMAT = "yyyy-MM-dd";
-
-  /** Content type for request. */
-  private static final String PROTOCOL_CONTENT_TYPE =
-    String.format("application/json; charset=%s", PROTOCOL_CHARSET);
-
-  private final Gson gson;
-  private final Class<T> clazz;
-  private final Map<String, String> headers;
-  private final Response.Listener<T> listener;
-  private final String requestBody;
-
-  /**
-   * Make a GET request and return a parsed object from JSON.
-   *
-   * @param url URL of the request to make
-   * @param clazz Relevant class object, for Gson's reflection
-   * @param headers Map of request headers
-   */
-  public GsonRequest(String url, Class<T> clazz, Map<String, String> headers,
+    private static final String LOG_TAG = GsonRequest.class.getCanonicalName();
```

Comment:
Did any of this code change?

Reply:
No, just fixed the spacing.  Our company uses 2 spaces for indentation and accidentally wrote some of these classes that way.